openmc/deplete/abc.py

```diff
@@ -612,6 +618,11 @@ class Integrator(ABC):
         Power density of the reactor in [W/gHM]. It is multiplied by
         initial heavy metal inventory to get total power if ``power``
         is not speficied.
+    timestep_units : {'s', 'min', 'h', 'd', 'MWd/kg'}
+        Units for values specified in the `timesteps` argument. 's' means
+        seconds, 'min' means minutes, 'h' means hours, and 'MWd/kg' indicates
+        that the values are given in burnup (MW-d of energy deposited per
+        kilogram of heavy metal).
```

Comment:
Just for clarity / explicitness 
```suggestion
        kilogram of initial heavy metal).
```